app/test_split_integer.py

```diff
@@ -3,7 +3,7 @@
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
     assert sum(split_integer(10, 4)) == 10, \
-        f"Expected sum of the parts "\
+        "Expected sum of the parts "\
         f"to be equal 10, but got {sum(split_integer(10, 4))}."
```

Comment:
remove this weird symbol at the end of all lines(`\`)

Reply:
pycharm automatically puts them. but is it better with parentheses?

Reply:
yup it is